invisible_cities/types/ic_types.py

```diff
@@ -74,7 +74,7 @@ def interval(self): return (self.min, self.max)
     def center(self): return (self.max + self.min) / 2
```

Comment:
I'm afraid I don't understand the comment of the commit... This change affects wherever in the code this method of the `minmax` class is used, even if the cython implementation disappears, right?

Reply:
The thing is that this class was not being used in favor of the cython implementation. There was this subtle difference with respect to the cython implementation that I had to fix in order to migrate all the code to use the python implementation.